howto/install/deploy-bare-metal.md

```diff
@@ -0,0 +1,93 @@
+To deploy Anbox Cloud on a public cloud (such as AWS, Azure or Google) or using MAAS or OpenStack, see the instructions in [Deploy Anbox Cloud with Juju](https://discourse.ubuntu.com/t/deploy-anbox-cloud-with-juju/17744).
```

Comment:
Do the instructions in that other document actually cover (or at least work for) MAAS or OpenStack?

Reply:
Yes, Juju supports MAAS and openstack as clouds, in the same way as actual cloud providers (GCP, AWS, etc), the instructions are similar